src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -159,4 +163,33 @@
 
     }
 
+    @AllArgsConstructor
+    @NoArgsConstructor
+    public static class IslandDamageSettings {
+
+        public boolean pvpOnIslands = true;
```

Comment:
I believe that this should be false by default because that's how it's working at the moment. Changing the default would cause confusion and unhappy players.

Reply:
plus skyblock is pve and enabling pvp on islands is dumb